docs/source/quick_start/install.rst

```diff
@@ -0,0 +1,199 @@
+------------------------------
+
+If you're using Windows, the command above should work fine, MCDR will be installed to global environment - you may ignore this section
+
+However, on Linux and Mac OS, if you're using a newer version of Python, you'll get something like this when you run pip install:
+
+.. code-block:: bash
+    
+    $ pip install mcdreforged
+    error: externally-managed-environment
+
+    × This environment is externally managed
+    ╰─> To install Python packages system-wide, try apt install
+        python3-xyz, where xyz is the package you are trying to
+        install.
+        
+        If you wish to install a non packaged Python package,
+        create a virtual environment using python3 -m venv path/to/venv.
+        Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
+        sure you have python3 full installed.
+        
+        If you wish to install a non packaged Python application,
+        it may be easiest to use pipx install xyz, which will manage a
+        virtual environment for you. Make sure you have pipx installed.
+        
+        See /usr/share/doc/python3.x/README.venv for more information.
+
+    note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
+    hint: See PEP 668 for the detailed specification.
+
+That's because of the `PEP 668 <https://peps.python.org/pep-0668/>`__, which was introduced to avoid pip conflicts with the system package manager (apt, yum, pacman, etc.)
+
+As workarounds, there're multiple options for you
+
+Using pipx
+~~~~~~~~~~
+
+Maybe the simplest solution for most users, but requires to use the third-party tool `pipx <https://pipx.pypa.io/>`__, which is designed to installing and running Python applications in isolated environments
+
+Install pipx refer to its `official documentation <https://pipx.pypa.io/stable/#install-pipx>`__
+
+Then install MCDR using pipx:
+
+.. code-block:: bash
+
+    $ pipx install mcdreforged
+
+When a new version of MCDR available, you may take the upgrade by:
+
+.. code-block:: bash
+
+    $ pipx upgrade mcdreforged
+
+.. asciinema:: resources/pipx.cast
+    :rows: 8
+
+|
+
+.. note::
+
+    In this way, MCDR will be installed in an isolated environment. Python packages required by MCDR plugins should be installed by:
+
+    * ``pipx inject mcdreforged <package_name>``
+    * or ``pipx inject mcdreforged -r requirements.txt``
+    
+    More conveniently, use the :ref:`\!!MCDR plg command <command/mcdr:Plugin management>` to install plugins with their dependencies
+
+
+Using virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the most native, but more complicated option, you can create a virtual environment and install MCDR in it
+
+Create a virtual environment, ``.venv`` for example, by:
+
+.. code-block:: bash
+
+    $ python3 -m venv .venv
```

Comment:
`venv` will be a better than `.venv`, cuz the `.` prefix is meaningless in windows. Also, hidden path might be confusing since it won't be listed in `ls -l`